packages/ui/src/components/containers/dialog/dialog.web.tsx

```diff
@@ -0,0 +1,96 @@
+import { JSXElementConstructor, ReactElement, ReactNode, cloneElement } from 'react';
+
+import { pxStringToNumber } from '@leather-wallet/utils';
+import * as RadixDialog from '@radix-ui/react-dialog';
+import { css } from 'leather-styles/css';
+import { Box } from 'leather-styles/jsx';
+import { token } from 'leather-styles/tokens';
+
+export interface DialogProps {
+  isShowing: boolean;
+  onClose?(): void;
+}
+interface RadixDialogProps extends DialogProps {
+  children: ReactNode;
+  footer?: ReactNode;
+  header?: ReactElement<any, string | JSXElementConstructor<any>>;
+  onGoBack?(): void;
+  wrapChildren?: boolean;
+}
+
+function getHeightOffset(header: ReactNode, footer: ReactNode) {
+  const headerHeight = header ? pxStringToNumber(token('sizes.headerHeight')) : 0;
+  const footerHeight = footer ? pxStringToNumber(token('sizes.footerHeight')) : 0;
+  return headerHeight + footerHeight;
+}
+
+function getContentMaxHeight(maxHeightOffset: number) {
+  const virtualHeight = window.innerWidth <= pxStringToNumber(token('sizes.popupWidth')) ? 100 : 70;
+
+  return `calc(${virtualHeight}vh - ${maxHeightOffset}px)`;
+}
+
+export function Dialog({
+  children,
+  footer,
+  header,
+  onClose,
+  isShowing,
+  wrapChildren = true,
+}: RadixDialogProps) {
+  if (!isShowing) return null;
+
+  const maxHeightOffset = getHeightOffset(header, footer);
+  const contentMaxHeight = getContentMaxHeight(maxHeightOffset);
+
+  return (
+    <RadixDialog.Root open>
+      <RadixDialog.Portal>
+        <RadixDialog.Overlay
+          className={css({
+            bg: 'overlay',
+            position: 'fixed',
+            inset: 0,
+          })}
+        >
+          <RadixDialog.Content
+            onPointerDownOutside={onClose}
+            className={css({
+              bg: 'ink.background-primary',
+              // remove borderRadius on small to give impression of full page
+              borderRadius: { base: '0', md: 'md' },
+              boxShadow:
+                'hsl(206 22% 7% / 35%) 0 10px 38px -10px, hsl(206 22% 7% / 20%) 0 10px 20px -15px',
+              position: 'fixed',
+              top: '50%',
+              left: '50%',
+              transform: 'translate(-50%, -50%)',
+              width: { base: '100vw', md: '90vw' },
+              height: { base: '100%', md: 'auto' },
+              maxWidth: { base: '100vw', md: 'pageWidth' },
+              maxHeight: { base: '100vh', md: '90vh' },
```

Comment:
I believe we discussed it in slack... 
isn't it better to add animations in keyframes rather than delete them at all? 😕

Reply:
I deleted them as I think they were never used and just mistakenly taken from the radix demo. 

I can add them into keyframes